CONTRIBUTING.md

```diff
@@ -175,3 +222,152 @@ Here are some examples of common test patterns:
     }
     ```
     Here we are using the `assert_empty` helper command that works with both Helm 2 and 3.
+    
+### Writing Acceptance Tests
+
+If you are adding a feature that fits thematically with one of the existing test suites,
+then you need to add your test cases to the existing test files.
+Otherwise, you will need to create a new test suite.
+
+We recommend to start by either copying the [example test](test/acceptance/tests/example/example_test.go)
+or the whole [example test suite](test/acceptance/tests/example),
+depending on the test you need to add.
+
+#### Adding Test Suites
```

Comment:
❤️

Reply:
This entire section is documentation gold.

Reply:
thanks!